packages/core-modules/contracts/storage/AssociatedSystemsStorage.sol

```diff
@@ -0,0 +1,23 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract AssociatedSystemsStorage {
+    struct AssociatedSystem {
+        address proxy;
+        address impl;
+        bytes32 kind;
+    }
+
+    struct AssociatedSystemsStore {
+        mapping(bytes32 => AssociatedSystem) satellites;
+    }
+
+    // solhint-disable-next-line func-name-mixedcase
+    function _associatedSystemsStore() internal pure returns (AssociatedSystemsStore storage store) {
+        assembly {
+            // bytes32(uint(keccak256("io.synthetix.associatedSystems")) - 1)
+            // todo
+            store.slot := 0x785a57f03c313e176889dae7a58117996eec43895df200b08edcd72475725cd5
```

Comment:
-1 missing: 0x785a57f03c313e176889dae7a58117996eec43895df200b08edcd72475725cd4

Reply:
ooc why do we do the -1? does it have any use?

Reply:
the idea behind the -1 is that the result is less likely to be the hash of any string as suggested in EIP-1967